### ISSUE TITLE
I'm might crash Newawe

### ISSUE THREAD

**Issue by joker314** [Newawe/newawe#61]
Not exactly an issue (yet), but this is the easiest way to let everybody know. I am about to type this
`<<<XSS>>>`
I did something similar, and I must admit it crashed the server. But I saw my code was working, and so it should not crash, and instead, take me to an error page I made. I will close this "issue" after the bug is fixed, but just wanted to let you know immediately. In a way, this is an issue, about whether we are susceptible XSS (cross-site scripting [attack]). If the project crashes, restarting should do the trick. Ok, I will now execute the code. I will set this to high-priority if I crash the server, ut for now, this is low priority. If this works, we can feel a bit safer about XSS threats, but we aren't exactly doing this for all user-inputs. 

tl;dr:Ignore this

**Comment by bjskistad** [comment]
K.

**Comment by joker314** [comment]
It worked!

**Comment by bjskistad** [comment]
Yay! (What does that mean, it stayed up, or crashed?)

**Comment by Radical-from-Scratch** [comment]
That is quite a good point

**Comment by bjskistad** [comment]
XP

**Comment by joker314** [comment]
It gave me a user-friendly error message, i.e. it did not crash. I say that it was friendly, but I have just amde a PR for a style in main.css to have something to make error messages better. Also, the error didn't start with a capital letter, I'll fix that in a sec. Am I allowed to merge directly since I will just be changing a single letter to capital?

**Comment by bjskistad** [comment]
I'll check em out.

**Comment by bjskistad** [comment]
Close?

**Comment by joker314** [comment]
Sure.
